recipe/run_test.py

```diff
@@ -23,10 +23,15 @@
     print("No GPU available. Exiting without running CuPy's tests.")
     sys.exit(0)
 
-# Print CuPy runtime info
-cupy.show_config()
```

Comment:
Can you move this inside `try`? This can be the test to check if a GPU is available or not.

Reply:
Sure, see d44f76c.